infra/blueprint-test/pkg/discovery/config.go

```diff
@@ -0,0 +1,64 @@
+package discovery
+
+import (
+	"fmt"
+	"os"
+
+	"sigs.k8s.io/kustomize/kyaml/yaml"
+)
+
+const (
+	DefaultTestConfigFilename = "test.yaml"
+	blueprintTestKind         = "BlueprintTest"
+	blueprintTestAPIVersion   = "blueprints.cloud.google.com/v1alpha1"
+)
+
+type BlueprintTestConfig struct {
+	yaml.ResourceMeta `json:",inline" yaml:",inline"`
+	Spec              struct {
+		Skip bool `json:"skip" yaml:"skip"`
+	} `json:"spec" yaml:"spec"`
+	Path string
+}
+
+// ShouldSkipTest checks if a test should be skipped
+func (b BlueprintTestConfig) ShouldSkipTest() bool {
```

Comment:
This should be a method on BlueprintTest, not BlueprintTestConfig. The fact that it comes from yaml is an implementation detail.

Reply:
added to respective BlueprintTests